app/src/main/res/layout/main_activity.xml

```diff
@@ -43,6 +43,7 @@
 
         <FrameLayout
             android:id="@+id/main"
+            android:background="@color/splashscreen_background"
```

Comment:
Why is the background of MainActivity being set by splashscreen background? This does not expose it but cause more issues.